core/src/main/kotlin/org/evomaster/core/output/naming/TestCaseNamingStrategy.kt

```diff
@@ -26,9 +26,10 @@ abstract class TestCaseNamingStrategy(
 
     /**
      * @param individual containing information for the test about to be named
+     * @param nameTokens list to collect the identifiers which will be formatted into the test case name
      *
      * @return a String with extra information that will be included in the test name, regarding the EvaluatedIndividual
      */
-    protected abstract fun expandName(individual: EvaluatedIndividual<*>): String
+    protected abstract fun expandName(individual: EvaluatedIndividual<*>, nameTokens: MutableList<String>): String
```

Comment:
unless expecting side effects on `nameTokens`, this should be passed as `List`

Reply:
@arcuri82 per my understanding, `List` in Kotlin is immutable, so passing it as `List` will not allow the collection of the name tokens. Using the `List.plus` function does not seem an option as well since function parameters are also immutable in Kotlin